variables/development.yml

```diff
@@ -5,8 +5,8 @@ web_vm_type: production-concourse-web
 worker_vm_type: production-concourse-worker
 iaas_worker_vm_type: production-concourse-iaas-worker
 web_vm_extensions: [production-concourse-lb]
-worker_vm_extensions: [production-concourse-profile, meta-data-v2]
```

Comment:
Are you **sure** that the settings will still apply on Concourse VMs? I ask because these settings are the crux of making sure that BOSH builds take minutes rather than hours

Reply:
See https://github.com/cloud-gov/deploy-concourse/pull/235

Reply:
Yes.  The setting is configured when the VM is created.  To put it in place we either wait for a stemcell or recreate the vms (what I've been doing).

If we find that this is not working for some reason, the vm_extension configuration has a higher priority than the CPI configuration, I won't be removing the vm_extension from the cloud config until it is removed as a separate configuration in deployment manifests.

Reply:
This is the configuration at the CPI layer:

```
- type: replace
  path: /instance_groups/name=bosh/properties/aws/metadata_options?/http_tokens?
  value: required

- type: replace
  path: /instance_groups/name=bosh/properties/aws/metadata_options?/http_put_response_hop_limit?
  value: 2
  ```
  
They are the same two values set by https://github.com/cloud-gov/deploy-concourse/pull/235 only now this is happening for ALL vms